src/std/stxutil.ss

```diff
@@ -14,3 +16,41 @@ package: std
 (def (format-id ctx fmt . args)
   (datum->syntax ctx (string->symbol (apply format fmt (map stx-e args)))
                  (stx-source ctx)))
+
+#;(import
```

Comment:
?

Reply:
Please remove craft.

Reply:
Oops, cruft from porting. Removed. Thanks for noticing.